apollo-converters/moshi/build.gradle

```diff
@@ -3,11 +3,18 @@ apply plugin: 'java'
 targetCompatibility = JavaVersion.VERSION_1_7
 sourceCompatibility = JavaVersion.VERSION_1_7
 
+configurations {
```

Comment:
`compileOnly` is part of the java plugin applied to a gradle buildscript. Not all projects support it. 

For android libraries, `compileOnly` isn't available since we're not applying the java plugin. We have to use `provided` for those.

For our other modules, we apply the java plugin so `compileOnly` is used.

```
compile-only dependencies can now be declared for all Java projects using the Java Plugin
```

This line creates a new `compileOnly` configuration. Its main task is to update the test sourceSet’s compileClasspath and runtimeClasspath to include to the `compileOnly` provided dependencies. Otherwise, you can't run the converters test.